src/modules/ipops/doc/ipops_admin.xml

```diff
@@ -883,6 +883,93 @@ if(dns_query("test.com", "xyz"))
 
     </section>
 
+    <section id="ipops.f.ptr_query">
+      <title>
+        <function moreinfo="none">ptr_query(ip, pvid)</function>
+      </title>
+
+      <para>
+		  Store the hostname that correspond to an IP address (IPv4 or IPv6)
+		  in a pseudo-variable variable $ptrquery(pvid=>hostname).
+      </para>
+
+      <para>
+      <emphasis>Note: Results will be in in $ptrquery(pvid=>key).</emphasis>
+      key can be:
+      <itemizedlist>
+        <listitem>
+          <para>
+            <emphasis>addr</emphasis> - IP address
+          </para>
+        </listitem>
+        <listitem>
+          <para>
+            <emphasis>type</emphasis> - type of IP address
+          </para>
+        </listitem>
+        <listitem>
+          <para>
+            <emphasis>ipv4</emphasis> - 1 if IPv4 address
+          </para>
+        </listitem>
+        <listitem>
+          <para>
+            <emphasis>ipv6</emphasis> - 1 if IPv6 address
+          </para>
+        </listitem>
```

Comment:
For these i am not exactly sure their purpose. They were accesible before these commits. Are they for counting IPv4/6 in dns_query or just to check if at least one IPv4/6 is associated with hostname?